src/components/LandingPage/landingPage.scss

```diff
@@ -0,0 +1,228 @@
+/*
+* landingPage.scss 
+* 
+* Dahlia Frontend - Andela Simulations Challenge ©️ 2019
+* Andela Simulations Challenge ©️ 2019
+* https://github.com/andela/dahlia-ah-frontend
+*
+* SCSS Designed by TeamDahlia.
+*/
+
+$grey: #e5e5e5;
+$white: #ffffff;
+$black: #000008;
+
+/* Writing Caption */
+#writingCaption {
+  min-height:100%;
+  background-size:cover;
+  background:url(https://res.cloudinary.com/allebd/image/upload/v1567024807/dahlia/Rectangle_4.png) rgba(0,0,0,0.3);
```

Comment:
Add the css under a parent class in the component so it doesn't clash with other css classes

Reply:
Noted, Treated Accordingly